tests/models/test_documents.py

```diff
@@ -38,31 +38,67 @@ def test_document_exists_check(self, mock_api_client):
             Document("not_a_real_judgment", mock_api_client)
 
     def test_judgment_name(self, mock_api_client):
-        mock_api_client.get_judgment_name.return_value = (
-            "Test Judgment v Test Judgement"
-        )
+        mock_api_client.get_judgment_xml.return_value = """
+            <root xmlns:akn="http://docs.oasis-open.org/legaldocml/ns/akn/3.0">
+                <akn:FRBRname value="Test Claimant v Test Defendant"/>
+            </root>
+        """
 
         document = Document("test/1234", mock_api_client)
 
-        assert document.name == "Test Judgment v Test Judgement"
-        mock_api_client.get_judgment_name.assert_called_once_with("test/1234")
+        assert document.name == "Test Claimant v Test Defendant"
+        mock_api_client.get_judgment_xml.assert_called_once_with(
+            "test/1234", show_unpublished=True
```

Comment:
wary that getting the judgment name might automatically show unpublished judgment's names -- I assume this is caught elsewhere?

Reply:
im assuming we just trust that once we have got the document we are happy that we can use it and all of its data.